<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProjectDialog</class>
 <widget class="QDialog" name="CreateProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="locationLayout">
     <item>
      <widget class="QLabel" name="emptyProjectLabel">
       <property name="text">
        <string>Empty Project</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>35</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="projectNameLayout">
       <item>
        <widget class="QLabel" name="projectNameLabel">
         <property name="text">
          <string>Project name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="projectNameInput"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="locationLabel">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="locationLayoutInner">
       <item>
        <widget class="QLineEdit" name="locationInput"/>
       </item>
       <item>
        <widget class="QPushButton" name="locationBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../sore_crow.qrc">
           <normaloff>:/Icons/icons/ryan/search.png</normaloff>:/Icons/icons/ryan/search.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="sourceLayout">
     <item>
      <widget class="QLabel" name="sourcesLabel">
       <property name="text">
        <string>Sources:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="videoSourceLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="videosLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Videos found</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="videosList"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="createLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="createSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createBtn">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="statusBarLayout"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../sore_crow.qrc"/>
 </resources>
 <connections/>
</ui>
